azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/LoadBalancer.java

```diff
@@ -582,6 +651,8 @@
         UpdateStages.WithBackend,
         UpdateStages.WithLoadBalancingRule,
         UpdateStages.WithInternetFrontend,
+        UpdateStages.WithInternalFrontend,
+        UpdateStages.WithNetworkSubnet,
         UpdateStages.WithInboundNatRule,
         UpdateStages.WithInboundNatPool {
     }
```

Comment:
Had an internal discussion and this will be fixed. - merging to unblock other work